packages/contracts-bedrock/src/L2/SoulGasToken.sol

```diff
@@ -15,10 +15,13 @@ import { Constants } from "src/libraries/Constants.sol";
 contract SoulGasToken is ERC20Upgradeable, OwnableUpgradeable {
     /// @custom:storage-location erc7201:openzeppelin.storage.SoulGasToken
     struct SoulGasTokenStorage {
-        // _minters are be whitelist EOAs, only used when !IS_BACKED_BY_NATIVE
+        // _minters are whitelist EOAs, only used when !IS_BACKED_BY_NATIVE
         mapping(address => bool) _minters;
         // _burners are whitelist EOAs to burn/withdraw SoulGasToken
         mapping(address => bool) _burners;
+        // _allow_sgt_value are whitelist contracts to consume sgt as msg.value
+        // when IS_BACKED_BY_NATIVE
+        mapping(address => bool) _allow_sgt_value;
```

Comment:
I think the naming should follow the same convention of existing OP ones, e.g., `_allowSgtValue`.

Reply:
Done in 62f2ca073785cce7e94cc37065db4f858f06659e.